src/components/verification-grid/verification-grid.ts

```diff
@@ -672,6 +873,23 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
         this.handleHelpRequest();
         break;
       }
+
+      // We don't handle the de-selection case here because we have deemed that
+      // de-selection on keyup is more intuitive.
+      // However, we have to preventDefault on the keydown because in some
+      // browsers (e.g. Chrome), pressing escape while the page is loading will
+      // cause the browser to stop page load.
+      // Therefore, if the user starts selecting the grid tiles before the page
+      // load finishes, and presses escape to de-select tiles, we don't want to
+      // cancel the page load.
+      case ESCAPE_KEY: {
+        // If the user has nothing selected, we still allow canceling the page
+        // navigation by pressing escape.
+        if (this.currentSubSelection.length > 0) {
```

Comment:
this is technically correct, but I actually thing we should just be bad citizens here and always prevent default if we handle it. Can you imagine trying to track down this bug?